include/xgboost/data.h

```diff
@@ -554,5 +555,22 @@ inline BatchSet<EllpackPage> DMatrix::GetBatches(const BatchParam& param) {
 
 namespace dmlc {
 DMLC_DECLARE_TRAITS(is_pod, xgboost::Entry, true);
-}
+
+namespace serializer {
+
+template <>
+struct Handler<xgboost::Entry> {
```

Comment:
Note. We don't need explicit serializer for Entry class in little-endian CPUs, because we can simply dump the class byte-by-byte. Only on big endian CPUs we need explicit serializer.